tldw_Server_API/app/api/v1/API_Deps/auth_deps.py

```
@@ -830,11 +830,31 @@ async def check_rate_limit(
     """
     # In TEST_MODE, bypass rate limiting entirely for deterministic tests
     try:
-        if os.getenv("TEST_MODE", "").lower() == "true":
+        raw = (os.getenv("TEST_MODE", "") or os.getenv("TLDW_TEST_MODE", "")).strip().lower()
+        if raw in {"1", "true", "yes", "y", "on"}:
             return  # Skip enforcement in test environments
     except Exception:
         pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)